### PR TITLE
Add conversion of cartesian coordinates into WGS84 coordinate system.

### PR DESCRIPTION
**Pull request by trajaona**
This follows #798.

### CODE REVIEW

**Comment by bangerth** [comment]
Please run the astyle indentation script.

File: CTestCustom.ctest [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This file will be installed into the build directory and causes ctest to
```
Comment (bangerth) [inline CTestCustom.ctest]: I don't think you need this file, do you?

File: print_test_info.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (bangerth) [inline print_test_info.sh]: same here

File: include/aspect/utilities.h [hunk 3]
```diff
@@ -89,47 +98,6 @@ namespace aspect
...
-     * which is not designed for very high order (> 100) spherical harmonics computation.
-     * If you use spherical harmonics of a high order be sure to confirm the accuracy first.
-     * For more information, see:
-     * http://www.boost.org/doc/libs/1_49_0/libs/math/doc/sf_and_dist/html/math_toolkit/special/sf_poly/sph_harm.html
-     */
-    std::pair<double,double> real_spherical_harmonic( unsigned int l, //degree
-                                                      unsigned int m, //order
-                                                      double theta,   //colatitude (radians)
-                                                      double phi );   //longitude (radians)
-
```
Comment (bangerth) [inline include/aspect/utilities.h]: I think you deleted this by accident, right? Please talk to one of the four of us about how to recover this -- I suspect that there was a failed merge conflict.

File: CTestCustom.ctest (hunk 1 above)
Comment (trajaona) [inline CTestCustom.ctest]: No I do not need it. I know now how to avoid it.

**Comment by bangerth** [comment]
@trajaona -- you still need to run the indentation script. Can you ask someone for help if you need it?

Also, did you copy this function from somewhere else in the code? If so, we should remove it from there and in all of the places where it is called, just call the new function.

**Comment by trajaona** [comment]
Yes, this function is derived from two functions from adiabatic boundary plugin.

**Comment by bangerth** [comment]
OK, in that case it would be good to delete the original functions if possible, so that we don't end up with duplicated code.

**Comment by trajaona** [comment]
I have already removed the original function.

File: source/initial_conditions/adiabatic_boundary.cc [hunk 4]
```diff
@@ -119,16 +73,15 @@ namespace aspect
     double
     AdiabaticBoundary<dim>::initial_temperature (const Point<dim> &position) const
     {
-      const std::pair<double, double> lat_long = lat_long_from_xyz_WGS84(position);
-      const double depth                       = radius_WGS84(lat_long.first) - position.norm();
-      const double isotherm_depth              = get_isotherm_depth(lat_long.first, lat_long.second);
+      std_cxx11::array<double,dim> wcoord  = aspect::Utilities::WGS84_coordinates(position);
```
Comment (bangerth) [inline source/initial_conditions/adiabatic_boundary.cc]: can you please indent the equals sign of this line like all of the others?

**Comment by bangerth** [comment]
Ah, nice! Must have looked at an individual commit, not the whole pull request. Thanks for implementing this! I have only one extra, small comment.

**Comment by trajaona** [comment]
I have corrected the equal sign indent.

**Comment by bangerth** [comment]
OK to merge once the tester is happy.

**Comment by bangerth** [comment]
You need to fix the indentation again, see http://www.math.clemson.edu/~heister/aspect-astyler-logs/72e024c8b9315de5ffeaefee6ed117a82a32aa75/

**Comment by trajaona** [comment]
done.

**Comment by bangerth** [comment]
OK after tester is happy.

**Comment by trajaona** [comment]
I wasn't aware of the namespace changed. I have fixed it.

**Pull request merged**